assets/src/edit-story/components/form/radioGroup/index.js

```diff
@@ -13,102 +13,75 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /**
  * External dependencies
  */
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { rgba } from 'polished';
-import { v4 as uuidv4 } from 'uuid';
 import { useMemo, useRef } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
 
 /**
  * Internal dependencies
  */
-import { Radio as Unselected, RadioSelected } from '../../../icons';
-import { KEYBOARD_USER_SELECTOR } from '../../../utils/keyboardOnlyOutline';
+import { Headline, Radio, themeHelpers } from '../../../../design-system';
 import useRadioNavigation from '../shared/useRadioNavigation';
 
-const Selected = styled(RadioSelected)`
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.radio};
-`;
-
-const RadioButton = styled.label`
-  display: block;
-  min-height: 30px;
-  margin: 10px 0;
-`;
-
-const RADIO_SIZE = 24;
-const TEXT_OFFSET = 30;
-
-const Label = styled.span`
-  display: flex;
-  position: relative;
-  margin-bottom: 0;
-  cursor: pointer;
-  svg {
-    width: ${RADIO_SIZE}px;
-    height: ${RADIO_SIZE}px;
-    margin-right: ${TEXT_OFFSET - RADIO_SIZE}px;
-  }
+const VisuallyHiddenLabel = styled(Headline)`
```

Comment:
question - if this is visually hidden, why is the `Headline` necessary?

Reply:
yeah that's a great question. We probably don't need this and can just use a basic heading!